### PR TITLE
Backlog 13684

### PR DESCRIPTION
**Pull request by Yousria**
## JIRA

https://jira.jahia.org/browse/BACKLOG-13684

### CODE REVIEW

File: validator-samples/src/main/java/org/foo/modules/validators/CustomValidators.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import org.osgi.service.component.annotations.Component;
+
+import java.util.Collections;
+import java.util.Map;
+
+/**
+ *
+ * @author yousria
+ */
+@Component(service = JCRNodeValidatorDefinition.class, immediate = true)
```
Comment (tdraier) [inline validator-samples/src/main/java/org/foo/modules/validators/CustomValidators.java:36]: just remove the immediate=true if not needed

Comment (Yousria) [reply validator-samples/src/main/java/org/foo/modules/validators/CustomValidators.java:36]: Ok :)

File: validator-samples/src/main/java/org/foo/modules/validators/TestNodeValidator.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import org.jahia.services.content.decorator.validation.JCRNodeValidator;
+import org.hibernate.validator.constraints.Length;
+import org.hibernate.validator.constraints.NotEmpty;
+import org.osgi.service.component.annotations.Component;
+
+/**
+ *
+ * @author yousria
+ */
+@Component(immediate = true)
```
Comment (tdraier) [inline validator-samples/src/main/java/org/foo/modules/validators/TestNodeValidator.java:36]: sorry i missed that one, what is the `@Component` for ? validator are created on demand for each node to validate, i'm not even sure you can create a component with that ?

Comment (Yousria) [reply validator-samples/src/main/java/org/foo/modules/validators/TestNodeValidator.java:36]: thought it was mandatory but Ok I'll remove that

**Pull request merged**